lectures/french_rev.md

```diff
@@ -1,1031 +1,1026 @@
+
+    * the Revolutionaries learned about this theory from Adam Smith's 1776 book The Wealth of Nations
+     and other contemporary sources
+
+    * It shaped how the Revolutionaries issued paper money called assignats from 1789 to 1791 
+
+* a classical **gold**  or **silver** standard
+  
+    * Napoleon, who became head of government in 1799 used this theory to guide his monetary and fiscal policies
+
+* a classical inflation-tax theory of inflation in which Philip Cagan's  demand for money studied 
+in this lecture  {doc}`cagan_ree` is a key component
+
+   * This theory helps us explain French price level and money supply data from 1794 to 1797  s
+
+* a *legal restrictions*  or *financial repression* theory of the demand for real balances 
+ 
+    * the Twelve Members comprising the Committee of Public Safety who adminstered the Terror from June 1793 to July 1794 used this theory to guide their monetary policy 
+
+We use matplotlib to replicate several of the graphs that {cite}`sargent_velde1995` used to portray outcomes of these experiments 
+
+---
+
+
+
+## Data Sources
+
+This lecture uses data from three spreadsheets:
+
+  * datasets/fig_3.ods
+  * datasets/dette.xlsx
+  * datasets/assignat.xlsx
+
+```{code-cell} ipython3
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+plt.rcParams.update({'font.size': 12})
+```
+
+## Government Expenditures and Taxes Collected
+
+
+
+We'll start by using matplotlib to construct two graphs that will provide important historical context.
+
+
+
+```{code-cell} ipython3
+---
+mystnb:
+  figure:
+    caption: "Military Spending in Britain and France"
+    name: fig4
+---
+# Read the data from Excel file
+data2 = pd.read_excel('datasets/dette.xlsx', sheet_name='Militspe', usecols='M:X', skiprows=7, nrows=102, header=None)
+# French military spending, 1685-1789, in 1726 livres
+data4 = pd.read_excel('datasets/dette.xlsx', sheet_name='Militspe', usecols='D', skiprows=3, nrows=105, header=None).squeeze()
+years = range(1685, 1790)
+
+plt.figure()
+plt.plot(years, data4, '*-', linewidth=0.8)
+
+plt.plot(range(1689, 1791), data2.iloc[:, 4], linewidth=0.8)
+
+plt.gca().spines['top'].set_visible(False)
+plt.gca().spines['right'].set_visible(False)
+plt.gca().tick_params(labelsize=12)
+plt.xlim([1689, 1790])
+plt.xlabel('*: France')
+plt.ylabel('Millions of livres')
+plt.ylim([0, 475])
+
+plt.tight_layout()
+plt.show()
+
+#plt.savefig('frfinfig4.pdf', dpi=600)
+```
+
+During the 18th century, Britain and France fought four large wars.
+
+Britain won the first three wars and lost the fourth.
+
+Each  of those wars  produced surges in both countries government expenditures that each country somehow had to finance.
+
+Figure {numref}`fig4` shows surges in military expenditures in France (in blue) and Great Britain.
```

Comment:
@HumphreyYang can we update these references to be a bit more specific such as `fr_fig4`